content/en/docs/concepts/packages/_index.md

```diff
@@ -111,26 +110,26 @@ the precise origin of imported files.
 The other directories hold packages that are facsimiles, derivatives, or
 augmentations of external packages:
 
-- *pkg*: an imported external CUE package,
+- *pkg*: imported CUE packages that are defined externally,
 - *gen*: CUE generated from external definitions, such as protobuf or Go,
 - *usr*: user-defined constraints for the above two directories.
 
 These directories split files from the same package across different
-parallel hierarchies based on the origin of the content.
-But for all intent and purposes they should be seen as a single directory
+parallel hierarchies based on the origin of the content, but for all
+intent and purposes they should be seen as a single directory
 hierarchy.
 
 The `cue.mod/usr` directory is a bit special here.
 The `cue.mod/pkg` and `cue.mod/gen` directories are populated by the `cue` tool.
 The `cue.mod/usr` directory, on the other hand, holds user-defined
 constraints for the packages defined in the other directories.
 
-User-defined constraints can be used to fill gaps in generated constraints;
-as generation is not always a sure thing.
-They can also be used to enforce constraints on imported packages, for instance
-to enforce that a certain API feature is still provided or of the desired form.
-The `usr` directory allows for a cleaner organization compared to storing
-such user-defined constraints directly in the `cue`-managed directories.
+User-defined constraints can be used to fill gaps in generated constraints,
+as generation is not always a sure thing. They can also be used to enforce
+additional constraints on imported packages— for instance to enforce that a
```

Comment:
(Recognising this was not part of your change, we can absolutely leave this for a later PR. Only mentioning because we are making changes in this space, so seemed like a good opportunity to discuss)

Again, I'm not sure "imported packages" is actually correct here. I think a better phrase would be "dependencies" - because the key is that we are talking about packages that are outside of the main module.